site/guide/work-with-content-blocks.qmd

```diff
@@ -24,7 +24,7 @@ Test-driven block
 
 ## Add content blocks
 
-1. [Log in to the ValidMind UI]({{< var vm_url >}}).
+1. [Log in to the {{< var vm_platform >}}]({{< var vm_url >}}).
 
 2. In the left sidebar, click [**Model Inventory**]({{< var vm_url >}}/model-inventory).
```

Comment:
Not within the scope of this PR but it feels like these compound URLs with variable + URL path don't buy us anything and they are likely more prone of breakage. Let's see how these pan out in the future? If they do break, a good rule of thumb might be to say that either all of a URL should be a variable or none of it.